src/utils/wallet.js

```diff
@@ -360,7 +500,31 @@ const wallet = {
       store.dispatch(updateTx(tx, updatedBalanceMap, balances));
```

Comment:
Is it still working for the new facade as well?

Reply:
We are currently not emitting this event on the wallet-service facade, but we should add a check here to ignore those messages until we start receiving it as it would not work on the wallet-service facade and probably lead to a crash

Added the check in c06ac242063733107488d4e7aee3c69456811ab5

Reply:
Are we going to emit this event in the future? How are we handling tx updates?

Reply:
We are not handling tx updates for now, there is an issue for that on the project spreadsheet